app/controllers/application_controller.rb

```diff
@@ -1,3 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :load_popular_users
+
+  private
+
+  def load_popular_users
```

Comment:
Same with this. When you copy-and-paste from existing stuff, make sure you know what it does and if it's relevant to the current task/project.

And if you don't know what something does, _ask_